packages/zoe/src/contracts/priceAggregator.js

```diff
@@ -31,6 +31,8 @@ import '@agoric/ertp/src/types-ambient.js';
 
 export const INVITATION_MAKERS_DESC = 'oracle invitation';
 
+/** @typedef {ParsableNumber | Ratio} Result */
```

Comment:
`Result` isn't very distinctive. Does it escape from this module?  Is it something like a `PriceQuote`?

Reply:
It doesn't escape. The short name is used for parity with `priceAggregatorChainlink.js` (forthcoming) which has a different `Result`.